README.md

```diff
@@ -5,3 +5,183 @@ UCL API Main Repository
 We're an amazing project, so obviously we have tests :sparkles:  
 Make sure you have the requirements installed in your virtual environment, and then run:  
 `python manage.py test --testrunner 'roombookings.custom_test_runner.NoDbTestRunner'`
+
+## Setting Up and Building
+This will walk you through setting yourself up a local development environment. This is different to if you want to deploy this into production, but luckily we have an internal set of installation scripts for this. Therefore, this guide will not walk you through, for example, getting Gunicorn or Nginx running. This is simply getting an environment up that can be used to develop the API. Testing with Nginx and Gunicorn is done in staging prior to a production deployment.
+
+### Requirements
+We only support development under Linux. macOS is unofficially supported, but you are on your own when it comes to finding the right Oracle client libraries. All Windows developers can and should use Bash on Ubuntu on Windows under Windows 10 (Creators Update) for testing, as the environment is Ubuntu 16.04.2 LTS; this is exactly the distribution and version of Linux that we use on both our Staging and Production web servers.
+
+Note that since the Creators Update (which includes 16.04.2; if you have not upgraded from Ubuntu 14 then there are tutorials online to do this) we have experienced zero issues building and running under Bash on Ubuntu on Windows.
+
+### Install Dependencies
+```
+sudo apt-get -y install git curl libpq-dev libpq5 libpython3-dev \
+    python3 python3-pip python3-virtualenv python-virtualenv \
+    unzip virtualenv libaio1 build-essential libpcre3 \
+    libpcre3-dev wget sed \
+    redis-server
+```
+
+### Start Redis
+```
+sudo service redis-server start
+```
+
+### Set up Oracle
```

Comment:
Is that how you do it? I added them to `/etc/environment`: https://help.ubuntu.com/community/EnvironmentVariables#System-wide_environment_variables